auth/client/src/main/resources/schema/elytron-client-1_4.xsd

```diff
@@ -325,6 +327,81 @@
         </xsd:choice>
     </xsd:complexType>
 
+    <xsd:complexType name="ssh-key-location-type">
+        <xsd:annotation>
+            <xsd:documentation>
+                The location of a private ssh key to be used as credentials.
+            </xsd:documentation>
+        </xsd:annotation>
+        <!-- The Passphrase used to encode the private key if applicable -->
```

Comment:
This should be reworded to indicate that this is the passphrase used to decrypt the private key file locally if needed.